src/DeviceControl.php

```diff
@@ -40,51 +40,47 @@ class DeviceControl extends CommonDevice
 {
     protected static $forward_entity_to = ['Item_DeviceControl', 'Infocom'];
 
-    public static function getTypeName($nb = 0)
+    public static function getTypeName($nb = 0): string
     {
         return _n('Controller', 'Controllers', $nb);
     }
 
-
-    public function getAdditionalFields()
+    public function getAdditionalFields(): array
     {
-
         return array_merge(
             parent::getAdditionalFields(),
-            [['name'  => 'is_raid',
-                'label' => __('RAID'),
-                'type'  => 'bool'
-            ],
-                ['name'  => 'interfacetypes_id',
+            [
+                [
+                    'name'  => 'is_raid',
+                    'label' => __('RAID'),
+                    'type'  => 'bool'
+                ],
+                [
+                    'name'  => 'interfacetypes_id',
                     'label' => __('Interface'),
                     'type'  => 'dropdownValue'
                 ],
-                ['name'  => 'devicecontrolmodels_id',
+                [
+                    'name'  => 'devicecontrolmodels_id',
                     'label' => _n('Model', 'Models', 1),
                     'type'  => 'dropdownValue'
                 ],
-                ['name'  => 'none',
-                    'label' => RegisteredID::getTypeName(Session::getPluralNumber()) .
-                                        RegisteredID::showAddChildButtonForItemForm(
```

Comment:
drop of showAddChildButtonForItemForm ?

Reply:
A previous PR of mine removed the ability to have HTML in the label (#16976) for several reasons.
Instead, the `showAddChildButtonForItemForm` method is called as part of the field itself in the dropdown Twig template.